Model/QueuedTask.php

```diff
@@ -72,7 +72,7 @@ public function createSingletonJob($jobName, $data, $notBefore = null, $group =
 				'completed' => null,
 				'or' => array(
 					'fetched' => null,
-					'failed' => 0,
+					'failed >' => 0,
```

Comment:
I'll update.
But the logic does need to change - we were getting collisions against a currently-in-progress instance, which is bad. We only want to match against 'pending' ones.